src/Document.php

```diff
@@ -10,6 +10,10 @@
  */
 class Document
 {
+    const META_ATTR = '_meta';
```

Comment:
nitpicking: Is there a specific reason not to address these properties directly?